src/server/scripts/Northrend/VaultOfArchavon/boss_toravon.cpp

```diff
@@ -12,151 +12,172 @@
 enum Spells
 {
     // Toravon
-    SPELL_FREEZING_GROUND               = 72090,
-    SPELL_FROZEN_ORB                    = 72091,
-    SPELL_WHITEOUT                      = 72034,
-    SPELL_FROZEN_MALLET                 = 71993,
+    SPELL_FREEZING_GROUND = 72090,
+    SPELL_FROZEN_ORB = 72091,
+    SPELL_WHITEOUT = 72034,
+    SPELL_WHITEOUT_VISUAL = 72036,
+    SPELL_FROZEN_MALLET = 71993,
 
     // Frozen Orb
-    SPELL_FROZEN_ORB_DMG                = 72081,    // priodic dmg aura
-    SPELL_FROZEN_ORB_AURA               = 72067,    // make visible
+    SPELL_FROZEN_ORB_DMG = 72081,    // priodic dmg aura
+    SPELL_FROZEN_ORB_AURA = 72067,    // make visible
 
     // Frozen Orb Stalker
-    SPELL_FROZEN_ORB_STALKER_VISUAL     = 72094,
+    SPELL_FROZEN_ORB_STALKER_VISUAL = 72094,
+
+    // Whiteout GroundEffect NPC
+    NPC_WHITEOUT_GROUND_EFFECT = 500005,
 };
 
 enum Events
 {
-    EVENT_FREEZING_GROUND               = 1,
-    EVENT_FROZEN_ORB_STALKER            = 2,
-    EVENT_CHECK_SUMMONS                 = 3,
+    EVENT_FREEZING_GROUND = 1,
+    EVENT_FROZEN_ORB_STALKER = 2,
+    EVENT_CAST_WHITEOUT = 3,
+    EVENT_CAST_WHITEOUT_GROUND_EFFECT = 4,
+    EVENT_CHECK_SUMMONS = 5,
+
 };
 
 enum Misc
 {
-    NPC_FROZEN_ORB                      = 38456,
-    NPC_FROZEN_ORB_STALKER              = 38461,
+    NPC_FROZEN_ORB = 38456,
+    NPC_FROZEN_ORB_STALKER = 38461,
 };
 
 
 class boss_toravon : public CreatureScript
 {
-    public:
-        boss_toravon() : CreatureScript("boss_toravon") { }
+public:
```

Comment:
Shouldn't this be 1 tab in?

Reply:
i think code beautifier is not a good idea here... right ?

Reply:
I dont know what that means, sorry D:

Reply:
https://codebeautify.org/cpp-formatter-beautifier

Reply:
Maybe... I dont know, I am not super good at C++, I just started to learn...
Personally I use Visual Studios and it will fix right tabs when you code

Reply:
i using visual code only maybe is the time to switch for Visual